CHANGELOG

```diff
@@ -11,8 +11,14 @@
 Version 3.3.3 (development)
 ===========================
 
-More efficient non-conforming adaptive mesh refinement
-------------------------------------------------------
+More general and efficient mesh adaptivity
+------------------------------------------
+- Added support for PUMI, the Parallel Unstructured Mesh Infrastructure from
```

Comment:
Revise/update the PUMI section if you want.

Reply:
This looks great. Thank you.